README.md

```diff
@@ -51,7 +51,7 @@ To start development run:
 
 -   `make init`: The first time you want to run the project locally
 -   `make dev-server`: Run the development server
--   `make dev-server-slave`: Run the development server slave that listens to a development server
+-   `make dev-module-client`: Run the development server slave that listens to a development server
```

Comment:
Do you even update the documentation bro?

Reply:
Fixed.